addons/advanced_fatigue/script_component.hpp

```diff
@@ -16,14 +16,28 @@
 
 #include "\z\ace\addons\main\script_macros.hpp"
 
+#define UNDERWEAR_WEIGHT 3.5
+
 #define ANTPERCENT 0.8
 #define SIM_BODYMASS 70
 #define JOULES_PER_ML_O2 20.9
 #define VO2MAX_STRENGTH 4.1
-#define REE 18.83 //((0.5617 * SIM_BODYMASS + 42.57) * 0.23)
-#define OXYGEN 0.9
-#define WATTSPERATP 7
+#define BIOMECH_EFFICIENCY 0.23
```

Comment:
If I'm reading this correctly, this is just replacing the constant 0.23 with a macro everywhere?

Reply:
Pretty much.